app/code/Magento/Wishlist/CustomerData/Wishlist.php

```diff
@@ -80,27 +80,13 @@ public function getSectionData()
     }
 
     /**
+     * Get number of Wishlist items
+     *
      * @return string
      */
     protected function getCounter()
     {
-        return $this->createCounter($this->wishlistHelper->getItemCount());
-    }
-
-    /**
-     * Create button label based on wishlist item quantity
-     *
-     * @param int $count
-     * @return \Magento\Framework\Phrase|null
-     */
-    protected function createCounter($count)
-    {
-        if ($count > 1) {
-            return __('%1 items', $count);
-        } elseif ($count == 1) {
-            return __('1 item');
-        }
-        return null;
+        return $this->wishlistHelper->getItemCount();
```

Comment:
@ptylek  We could return here a string so not to fix PAT scenario
```suggestion
        return (string) $this->wishlistHelper->getItemCount();
```
What do you think?

Reply:
@slavvka thank you for checking this! 
Well, function's name `getItemCount` suggests that it returns number so it might be confusing when it returns string. If you think that returning a string can be justified in this case (maybe adding a comment that explains why the change was introduced), I can adjust this PR (so it compares to string and not number). I'm not very familiar with sanity checks so I'd rather take your recommendation regarding how to proceed.

Reply:
You are absolutely right that it could be confusing. Currently it should return a string according to PHPDoc, though:
```
@return string
```

Efforts needed to change PAT sanity checks will require PRs to CE, EE, B2B and Infra. I am not sure if the fix is worth it. 

Anyway we could try it and see if sanity check passes with it. If no we will need to male all those changes.

Reply:
I think that the conversion to string will occur anyway on the front so this workaround could be acceptable.